1.Fuzzing_technology/README.md

```diff
@@ -285,13 +318,18 @@ Your task is to do the following:
 
 ## Task 4
 
-### Creating your own fuzzer and fuzzing it
+### Creating your own fuzzer and fuzzing with it
+
+This task is to be completed in a Jupyter notebook.
+
+You can access an online notebook following this link: https://notebooks.rahtiapp.fi/welcome TODO?
 
-This task is to be completed in a jupyter notebook. You can access an online notebook following this link: https://notebooks.rahtiapp.fi/welcome
+Use your university credentials 'Haka' to log in.
+Select the 'Introduction to Python' notebook to work with. Do note that these notebooks only have a lifetime of 4 hours, so make sure to download and save your work!
 
-Use your university credentials 'Haka' to login. Select 'Introduction to Python' notebook to work with. Do note that these notebooks only have a lifetime of 4 hours, so make sure to download and save your work! Attach your notebook file ```lab1_fuzzer_your_name.ipynb``` as a return in moodle return_box for Lab-1 alongside this report.
+Attach your notebook file ```lab1_fuzzer_your_name.ipynb``` as a return in moodle return_box for Lab-1 alongside this report. TODO?
```

Comment:
This should be returned to GitHub. GitHub can render .ipynb files directly and are then easy to review. Students need to run code in the book before returning it.